plugin/src/main/java/net/dzikoysk/funnyguilds/data/database/Database.java

```diff
@@ -1,17 +1,27 @@
 package net.dzikoysk.funnyguilds.data.database;
 
 import com.zaxxer.hikari.HikariDataSource;
-import java.sql.Connection;
-import java.sql.SQLException;
 import net.dzikoysk.funnyguilds.FunnyGuilds;
 import net.dzikoysk.funnyguilds.config.sections.MysqlConfiguration;
 import net.dzikoysk.funnyguilds.shared.FunnyStringUtils;
 
+import java.sql.Connection;
+import java.sql.SQLException;
+
 public class Database {
 
     private final HikariDataSource dataSource;
 
     public Database() {
+        String jdbcClassName = FunnyGuilds.getInstance().getPluginConfiguration().dataModel.getJDBCClassName();
+        if (jdbcClassName != null) {
```

Comment:
Wtedy chyba też by wypadało dać jakiś błąd - jeśli tu kod dotarł, a nie ma żadnej klasy, to chyba jest coś nie tak

Reply:
Nie widziałem żadnego if'a który tworzy konstruktor Database, że DataModel nie jest FLAT

Reply:
?